src/Nethermind/Nethermind.Runner/NLog.config

```diff
@@ -56,13 +56,13 @@
     <logger name="JsonWebAPI*" minlevel="Error" writeTo="auto-colored-console-async" final="true" />
     <logger name="JsonWebAPI*" final="true" />
 	
-	<!-- <logger name="Merge.Plugin.PoSSwitcher" minlevel="Trace" writeTo="file-async"/> -->
- <!--    <logger name="Merge.Plugin.PoSSwitcher" minlevel="Trace" writeTo="auto-colored-console-async"/> -->
- <!--    <logger name="Merge.Plugin.PoSSwitcher" final="true"/> -->
- <!-- -->
- <!--    <logger name="Merge.Plugin.Synchronization.*" minlevel="Trace" writeTo="file-async"/> -->
- <!--    <logger name="Merge.Plugin.Synchronization.*" minlevel="Trace" writeTo="auto-colored-console-async"/> -->
- <!--    <logger name="Merge.Plugin.Synchronization.*" final="true"/> -->
+	<logger name="Merge.Plugin.PoSSwitcher" minlevel="Trace" writeTo="file-async"/>
+    <logger name="Merge.Plugin.PoSSwitcher" minlevel="Trace" writeTo="auto-colored-console-async"/>
```

Comment:
NLog to revert, besides, I have no objections

Reply:
yes, yes I know :) that is why I drafted PR, but I'm running hive with extended NLog :)